Client/src/components/profile/profile.tsx

```diff
@@ -0,0 +1,180 @@
+                  onChange={changeUsername}
+                  className="input-body"
+                  maxLength={10}
+                />
+                <button
+                  id="change-username-profile"
+                  className="input-button"
+                  onClick={(e) => {
+                    updateChanges(e);
+                    setInputOpen(false);
+                  }}
+                >
+                  Save
+                </button>
+              </form>
+              <h2>Email: {userRedux.email}</h2>
+            </div>
+          </div>
+          <div className="learning-progress">
+            <h1 className="learning-progress">Learning progress:</h1>
+            <div className="progress">
+              <div className="sort-progress">
+                <div>
+                  <img className="sort-progress-img" src={sort} />
+                </div>
+                <h3 className="algo-name">Sorting Algorithms</h3>
+                <h3>{userRedux.sortLessons}</h3>
+                <h3 className="progress-number">{currentSortProgress}/6</h3>
+              </div>
+              <div className="path-progress">
+                <div>
+                  <img className="path-progress-img" src={path} />
+                </div>
+                <h3 className="algo-name">Path Finding Algorithms</h3>
+                <h3>{userRedux.pathLessons}</h3>
+                <h3 className="progress-number">{currentPathProgress}/4</h3>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className="user-stats">
+          <h1 className="achievements">ACHIEVEMENTS</h1>
+          <div className="circles">
+            <div
+              className="circle"
+              onClick={() => alert('are you circle?')}
+            ></div>
+            <div
+              className="circle"
+              onClick={() => alert('are you circle?')}
+            ></div>
+            <div
+              className="circle"
+              onClick={() => alert('are you circle?')}
+            ></div>
+            <div
+              className="circle"
+              onClick={() => alert('are you circle?')}
+            ></div>
+          </div>
+          <div className="stats-line">
+            <h4 className="stats-text">
+              MATCHES
+              <h4 className="stats-text">14</h4>
+            </h4>
+            <h4 className="stats-text">
+              WINS<h4 className="wins-yellow">3</h4>
+            </h4>
+            <h4 className="stats-text">
+              LOSSES<h4 className="losses-red">2</h4>
+            </h4>
+            <h4 className="stats-text">
+              WIN RATE<h4 className="stats-text">60%</h4>
+            </h4>
+            <h4 className="stats-text">
+              GOLD EARNED<h4 className="stats-text">200</h4>
+            </h4>
+          </div>
+          <h3 className="created">
+            Account created {new Date().toLocaleDateString()}
```

Comment:
Change date to when the account was created, not current one (ask Josep about the data from Auth0)